test/lib/jdk/test/lib/Asserts.java

```diff
@@ -234,59 +234,58 @@ public static void assertSame(Object lhs, Object rhs, String msg) {
     }
 
     /**
-     * Asserts that {@code lhs} is the same byte array as {@code rhs}.
+     * Asserts that {@code actual} is the same byte array as {@code expected}.
      *
-     * @param lhs The left hand side of the comparison.
-     * @param rhs The right hand side of the comparison.
+     * @param expected The expected value
+     * @param actual The actual value
      * @throws RuntimeException if the assertion is not true.
      * @see #assertEqualsByteArray(byte[], byte[], String)
      */
-    public static void assertEqualsByteArray(byte[] lhs, byte[] rhs) {
-        assertEqualsByteArray(lhs, rhs, null);
+    public static void assertEqualsByteArray(byte[] expected, byte[] actual) {
```

Comment:
Based on [ML_KEM_Test.java](https://java.se.oracle.com/source/xref/jdk-jdk/jdk-open/test/jdk/sun/security/provider/acvp/ML_KEM_Test.java) and [ML_DSA_Test.java](https://java.se.oracle.com/source/xref/jdk-jdk/jdk-open/test/jdk/sun/security/provider/acvp/ML_DSA_Test.java), it looks like the first argument should be actual and the 2nd argument is expected.

Reply:
Also, it's not about "the same byte array", but more like "the same bytes" or have equal values, right?

Reply:
Correct, I'll fix.

As for its usages (in `ML_KEM_Test.java`), I'll be happy to fix them. In fact, I first noticed those inconsistency and then decided to make the changes here. Would you prefer them to be fixed in this PR?

Reply:
Yes, I'd prefer to fix it together.

Reply:
Sure, will do now.